lib/Quantization/Quantization.cpp

```diff
@@ -432,6 +432,7 @@ quantizeFunction(const ExecutionEngine &EE,
     }
   } while (nodeIt != stopIt);
 
+  F->getParent()->eraseFunction(F);
```

Comment:
Please add a comment here explaining why we erase the old function.

Reply:
all done